src/cli/config/index.test.ts

```diff
@@ -1,25 +1,28 @@
-const { resolve } = require('path');
+import { resolve, join } from 'path';
 
-const {
+import {
   configDefault,
   loadConfig,
   loadJson5,
   resolveModuleOrPath,
   searchPlaces,
   search,
-} = require('./config');
+} from '.';
+
+const rootDir = resolve(__dirname, '../../../');
```

Comment:
With `tsconfig.json` thoughts on using `paths: {}` and [`tsconfig-paths`](https://www.npmjs.com/package/tsconfig-paths) to discourage the use of `../` for `@src/some-path`?

Reply:
Not opposed. I go either way since I think it _can_ make it harder for new contributors, but it makes writing nicer.